actions/plugins/publish/publish/publish.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -e
+if [ $RUNNER_DEBUG == "1" ]; then
```

Comment:
This syntax is causing errors:
```
/opt/actions-runner/_work/_actions/grafana/plugin-ci-workflows/main/actions/plugins/publish/publish/publish.sh: line 3: [: ==: unary operator expected
```

Should probably be:

```suggestion
if [ "$RUNNER_DEBUG" == "1" ]; then
```

Reply:
Great catch! https://github.com/grafana/plugin-ci-workflows/pull/86